### PR TITLE
`-main`: accept an optional classpath argument

### PR DESCRIPTION
**Pull request by vemv**
By accepting a fixed user-provided string as the classpath, one can be sure that lein-nvd is not interfering in classpath computation and therefore one prevents false positives/negatives.

Fixes https://github.com/rm-hull/lein-nvd/issues/46

Also gives a way to easily solve:

https://github.com/rm-hull/lein-nvd/issues/50
https://github.com/rm-hull/lein-nvd/issues/73
https://github.com/rm-hull/lein-nvd/issues/74

### CODE REVIEW

**Review by vemv — COMMENTED** [review]
Amended with some refinements and an `assert` (as a lightweight test)

**Review by rm-hull — COMMENTED** [review]
Looks fine 👍 
I'll investigate the failure on master

**Comment by rm-hull** [comment]
Since travis has gone all shit, this project really could do with moving to github actions

**Comment by vemv** [comment]
Alright!

Feel free to LMK when `master` is green again. Meanwhile I'll try to add a simple test (I think a bash script would be nice for exercising Lein + deps.edn + the new argument all in the sample place)

**Comment by rm-hull** [comment]
Main branch is now green

**Comment by vemv** [comment]
Looks like master went red again after https://github.com/rm-hull/lein-nvd/commit/f4a7187a818225c03b89f99370cd3bcd6d5a3971

**Comment by vemv** [comment]
Finally got the build green 😄 (with the caveat of https://github.com/rm-hull/lein-nvd/pull/75#issuecomment-839277412)

Also, I used the occasion to introduce a JVM matrix in CI, so that we're sure lein-nvd runs correctly in all JVMs that Clojure officially supports (today: 8 and 11).

Similarly, the integration test script runs in a parallel job, for a faster feedback loop.

File: test/nvd/task/check_test.clj [hunk 1]
```diff
@@ -44,10 +44,10 @@
   (update-db/-main "test/resources/self-test.json")
   (let [project (check/-main "test/resources/self-test.json")]
     (is (== 11.0 (get-in project [:nvd :fail-threshold])))
-    (is (== 5.0 (get-in project [:nvd :highest-score])))
+    (is (== 0 (get-in project [:nvd :highest-score])))
```
Comment (rm-hull) [inline test/nvd/task/check_test.clj:47]: slightly puzzled why this dropped from 7.5 to 5.0 and now to zero .. happy to take this for now, but think i will create an issue to update `self-test.json` so we have something that is > 0

**Review by rm-hull — APPROVED** [review]

**Pull request merged**